src/test/clojure/xtdb/node_test.clj

```diff
@@ -917,3 +917,10 @@ VALUES(1, OBJECT (foo: OBJECT(bibble: true), bar: OBJECT(baz: 1001)))"]])
   (t/is (thrown-with-msg? IllegalArgumentException
                           #"Port value out of range: 99999"
                           (xtn/start-node {:server {:port 99999}}))))
+
+
+(t/deftest backfill-test
```

Comment:
doesn't look like it's testing anything?

Reply:
Yep, leftover.